pcdsdevices/smarpod.py

```diff
@@ -0,0 +1,133 @@
+"""
+Module for the SmarPod and related devices.
+"""
+from ophyd.device import Component as Cpt
+from ophyd.device import Device
+from ophyd.signal import EpicsSignal, EpicsSignalRO
+
+
+class SmarPodPose(Device):
```

Comment:
Out of the loop question: What exactly does `Pose` mean in this context? I think some class level docstrings would make this file more friendly to people like me who don't know what a smarpod is.

e.g. something like this but with correct info
```
"""
SmarPodPose is a class that controls a smarpod positioner.

It is included in the more general SmarPod object, which represents a full controller.
A smarpod is a hexapod-like 6-axis positioner.
"""
```

Onto each of the classes to just state what they are.